src/project_manager/ros_utils.cpp

```diff
@@ -706,6 +761,28 @@ Utils::FileName ROSUtils::getCatkinToolsProfilesYamlFile(const Utils::FileName &
     return profiles;
 }
 
+bool ROSUtils::setCatkinToolsProfilesYamlFile(const Utils::FileName &workspaceDir,
```

Comment:
I had to change it since calling setCatkinToolsActiveProfile with profileName would create "default" file  if proifileName was set to "default" and Yaml file was missing. I rested this but let me know if you see anything else.